### PR TITLE
js: refactoring fraction validation page

### PR DESCRIPTION
**Pull request by mgenov**
The page that validates fractions is now refactored and is having tests (test.html)

isFraction function is extracted from the validate() function to provide better design and easy testing.

### CODE REVIEW

File: fractionvalidator/test.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Fraction Tests</title>
+	</head>
+	<body>
+		<script src="https://unpkg.com/expect/umd/expect.min.js"></script>
```
Comment (denisdimitrov) [inline fractionvalidator/test.html]: Is this the required lib for the JS testing?

Comment (mgenov) [inline fractionvalidator/test.html]: No, this is the easiest one that I found.

Comment (mgenov) [inline fractionvalidator/test.html]: Looks simple and is good for learning purposes.

Comment (denisdimitrov) [inline fractionvalidator/test.html]: I see, I will try it myself. Thank you, this is very helpful.

**Pull request closed without merging**